indexer/src/constants.ts

```diff
@@ -1,52 +1,28 @@
 import { padString } from "./utils/hex.ts";
 import { hash } from "./deps.ts";
 
-// Get Sink Type or returns "console" if the value is null or undefined
-export const SINK_TYPE = (() => {
-  const addr = Deno.env.get("SINK_TYPE") ?? "console";
-  if (addr !== "console" && addr !== "mongo") {
-    throw new Error("Invalid SINK_TYPE");
+const getEnvVariable = <T>(key: string, defaultValue: T, validator?: (value: T) => boolean): T => {
+  const value = Deno.env.get(key) as T ?? defaultValue;
+  if (validator && !validator(value)) {
+    throw new Error(`Invalid ${key}`);
   }
-  return addr;
-})();
+  return value;
+};
 
-// Get the sink options from the sink type
-export const SINK_OPTIONS = SINK_TYPE === "mongo"
-  ? {
-    connectionString: Deno.env.get("MONGO_CONNECTION_STRING") ??
-      "mongodb://mongo:mongo@mongo:27017",
-    database: Deno.env.get("MONGO_DATABASE_NAME") ?? "kakarot-test-db",
-    collectionNames: ["headers", "transactions", "receipts", "logs"],
-  }
-  : {};
-
-// Get the starting block or returns 0 if the value is null or undefined
-export const STARTING_BLOCK = (() => {
-  const startingBlock = Number(Deno.env.get("STARTING_BLOCK") ?? 0);
-  return Number.isSafeInteger(startingBlock) && startingBlock >= 0
-    ? startingBlock
-    : (() => {
-      throw new Error("Invalid STARTING_BLOCK");
-    })();
-})();
-
-// Get authentication token from Apibara or returns an empty string if the value is null or undefined
-export const AUTH_TOKEN = Deno.env.get("APIBARA_AUTH_TOKEN") ?? "";
+export const SINK_TYPE = getEnvVariable("SINK_TYPE", "console", (value) => ["console", "mongo"].includes(value as string));
 
-// Get stream URL or returns "http://localhost:7171" if the value is null or undefined
-export const STREAM_URL = Deno.env.get("STREAM_URL") ?? "http://localhost:7171";
+export const SINK_OPTIONS = SINK_TYPE === "mongo" ? {
+  connectionString: getEnvVariable("MONGO_CONNECTION_STRING", "mongodb://mongo:mongo@mongo:27017"),
+  database: getEnvVariable("MONGO_DATABASE_NAME", "kakarot-test-db"),
+  collectionNames: ["headers", "transactions", "receipts", "logs"],
+} : {};
```

Comment:
I think you might need to run Deno formatting

Reply:
ok doing

Reply:
> I think you might need to run Deno formatting

done!